.travis.yml

```diff
@@ -1,10 +1,19 @@
 language: bash
+
 os:
 - linux
 - osx
 
+services:
+  - docker
+
 script:
-- bash -c 'time bin/bats --tap test'
+- |
+  bash -c 'time bin/bats --tap test'
```

Comment:
Don't forget the `set -e`!

Reply:
I've added it but didn't think it was necessary (not much of a Travis user TBH), although fail fast is a good life rule :+1:  https://docs.travis-ci.com/user/customizing-the-build/#Customizing-the-Build-Step

Reply:
D'oh! I missed that detail. Go ahead and put it back like you had it, then I'll merge.

Reply:
Reverted that line, and interested to know how Travis do it if anybody's got insight. `set -e` with custom `trap` or similar? Doubt it's per-line execution, but who knows...

Reply:
I'm imagining they just execute each entry in the YAML list (via `eval` or `bash -c` or writing a temp file and running it) and set a `failed` flag if any one returns nonzero.